content/updates/2020-08/1-API-search.md

```diff
@@ -0,0 +1,22 @@
+---
+pim_announcement_img: ./img/api-filter-attributes.png
+pim_announcement_alt_img: "API documentation about attribute filters"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# New filters on product structure API endpoints
+::: meta-data type="New" features="API, Productivity" available="late-August" link-to-doc="https://api.akeneo.com/documentation/filter.html#"
+:::
+
+With the new `search` parameter, you can now add filters on product structure endpoints: attributes, attribute groups, categories, and families.  
+For example, you can retrieve data for all attributes that have been updated since a specific date, or you can get the data from all the sub-categories of a specific tree.
+
+::: more
+[How to filter attributes?](https://api.akeneo.com/documentation/filter.html#filter-attributes)
+[How to filter attribute groups?](https://api.akeneo.com/documentation/filter.html#filter-attribute-groups)
+[How to filter families?](https://api.akeneo.com/documentation/filter.html#filter-families)
+[How to filter categories?](https://api.akeneo.com/documentation/filter.html#filter-categories)
+[How to manage your connections?](../articles/manage-your-connections.html)    
```

Comment:
As you don't mention the connections at all in the news, it feels weird to have a link to the documentation about connections.

Reply:
I hesitated when I added the link about connections but you're right.